docs/plugindev/README.md

```diff
@@ -232,6 +232,34 @@ make inventory-init
 make inventory-plugin-add
 ```
 
+### Updating the plugin group
+
+Create `plugin-scope-association.yaml` under `$(ROOT_DIR)/cmd/plugins/plugin-scope-association.yaml` of plugin repository
+
+Sample file content
+
+```yaml
+plugins:
+- name: builder
+  target: global
+  isContextScoped: false
+- name: test
+  target: global
+  isContextScoped: false
+```
+
+Update the PLUGIN_SCOPE_ASSOCIATION_FILE in plugin-tooling.mk to file location
+
+``` makefile
+PLUGIN_SCOPE_ASSOCIATION_FILE ?= $(ROOT_DIR)/cmd/plugins/plugin-scope-association.yaml
+```
+
```

Comment:
Whenever a `plugin-scope-association.yaml` association changes, the plugins must be rebuilt to generate the proper plugin group information.  We need to mention this here.  The user must run `make plugin-build` again.

Reply:
updated the instructions